pkg/utils/utils.go

```diff
@@ -9,3 +9,15 @@ import (
 func IsGPUInstanceType(instanceType string) bool {
 	return strings.HasPrefix(instanceType, "p2") || strings.HasPrefix(instanceType, "p3")
 }
+
+func HasGPUInstanceType(instanceTypes []string) bool {
```

Comment:
Is there another place where we can use this function? Also, seems like it could actually take `ng *api.NodeGroup` instead, but we don't have to do that in this PR.

Reply:
Yeah, I thought about that. but didn't want to drag the dependency. I would put it in the api package, with the hasMixedInstances. WDYT?